nucypher/blockchain/eth/interfaces.py

```diff
@@ -226,27 +223,13 @@ def connect(self):
         # Connect
         web3_instance = Web3(provider=self.__provider)  # Instantiate Web3 object with provider
         self.w3 = web3_instance
+        self.client = Web3Client(self.w3)
```

Comment:
Perhaps this line at the web3 instantiation above can be combined into one line:

`self.client = Web3Client(Web3(provider=self.__provider))`